pkgs/by-name/sv/svlangserver/package.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildNpmPackage, fetchFromGitHub }:
+
+buildNpmPackage rec {
+  pname = "svlangserver";
+  version = "0.3.5";
+
+  src = fetchFromGitHub {
+    owner = "imc-trading";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-CkcKyC2W6NBvxkwYDVHpBF5T2NMiiDVVwR1mftEks54=";
+  };
+
+  npmDepsHash = "sha256-XovXh3weLh7xO96chlYwkqUnzganjLmKcUsAdBhA060=";
+
+  npmPackFlags = [ "--ignore-scripts" ];
+
+  meta = with lib; {
```

Comment:
`with lib;` is considered bad 

https://github.com/NixOS/nixpkgs/issues/208242